WTWSMS/localisation/WtWSMS_warfare.csv

```diff
@@ -26,7 +26,10 @@ RETTYPE_CUL_ROMAN;Legio Comitatensis;Legio Comitatensis;;;;;;;;;;;;x
 RETTYPE_CUL_ILLY;Balkan Falxmen;Guerriers balkaniques � falx;;;;;;;;;;;;x
 RETTYPE_CUL_RBT;Brito Bellatorum;Brito Bellatorum;;;;;;;;;;;;x
 RETTYPE_CUL_GMR;Germanorum Peditum;Germanorum Peditum;;;Germanorum Peditum;;;;;;;;;x
-RETTYPE_CUL_PAR;Teutonicus Foederatus;Teutonicus Foederatus;;;Federados Teut�nicos;;;;;;;;;x
+#RETTYPE_CUL_PAR;Teutonicus Foederatus;Teutonicus Foederatus;;;Federados Teut�nicos;;;;;;;;;x
```

Comment:
This can be removed.

Reply:
I thought we could keep the name for some future retinue but OK, removed.